src/doublyLinkedList.js

```diff
@@ -0,0 +1,198 @@
+'use strict'
+
+function Node(data) {
+  this.previous = null
+  this.data = data
+  this.next = null
+}
+
+export default class DoublyLinkedList {
+  constructor() {
+    this.length = 0
+    this.head = null
+  }
+
+  isEmpty() {
+    return this.length === 0
+  }
+
+  size() {
+    if (this.length === 0){
+      return null
+    }
+    return this.length
+  }
+
+  clear() {
+    this.length = 0
+    this.head = null
+  }
+
+  getHeadNode(data) {
```

Comment:
why does getHeadNode() have a parameter of data? isnt it just going to return the head ?? like it doesnt need an argument?